projects/OpenWebCalendar/default.nix

```
@@ -0,0 +1,16 @@
+{
+  pkgs,
+  lib,
+  sources,
+  ...
+}@args:
+{
+  metadata.subgrants = [ "OpenWebCalendar" ];
+  nixos.modules.services.open-web-calendar = {
+    name = "open-web-calendar";
+    module = "${sources.inputs.nixpkgs}/nixos/modules/services/web-apps/open-web-calendar.nix";
+    examples = { };
```

```suggestion
    examples.basic = null;
```

Let's communicate that we owe an example.

---

done